FlowCal/mef.py

```diff
@@ -149,30 +168,66 @@ def selection_std(populations,
     low, high : int or float
         Low and high thresholds. Required if the elements in `populations`
         are numpy arrays. If not specified, and the elements in
-        `populations` are FCSData objects, use 0.015 of the lowest value
-        and 0.0985 of the highest value in ``populations[0].range``.
+        `populations` are FCSData objects, use 1.5% and 98.5% of the range
+        in ``populations[0].range``.
     n_std_low, n_std_high : float, optional
         Number of standard deviations from `low` and `high`, respectively,
         that a population's mean has to be closer than to be discarded.
+    scale : str, optional
+        Rescaling applied to `populations` before calculating means and
+        standard deviations. If ``linear``, no rescaling is applied. If
+        ``log``, apply base-10 logarithm to `populations`.
```

Comment:
seems like `scale` could be generalized to just accept a function (perhaps something conforming to the `transform` module function signatures?) which is applied to `data`, `low`, and `high`.

makes me wonder why this needs to be passed to this function explicitly in the first place, though, instead of just performing the transformation on `data` before calling this function. (this is a deeper seated debate we've had before which is kinda reflected in the `mef.get_transform_fxn()` function, as well as a few other places I think)